package/com.unity.formats.usd/Dependencies/USD.NET.Unity/UnityTypeBindings.cs

```diff
@@ -55,10 +55,12 @@ static public void RegisterTypes()
                 (object obj) => UnityTypeConverter.Vector2ToVec2f((Vector2)obj),
                 (pxr.VtValue value) => UnityTypeConverter.Vec2fToVector2(value),
                 SdfValueTypeNames.Float2));
+            binder.typeAliases.Add(SdfValueTypeNames.TexCoord2f.GetAsToken(), SdfValueTypeNames.Float2.GetAsToken());
```

Comment:
It's odd that this API requires `GetAsToken`, can we do that internally, maybe via a wrapper API?

Reply:
Done in https://github.com/Unity-Technologies/usd-unity-sdk/pull/260/commits/4e5e928f3618110e5ec30616b01f6e413329a25c